src/website/templates/action_submitted_email.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
i'm guessing these emails are all in the postmark as well?

Reply:
If so in our email helper function can we do it such that while in production, the emails are sent from postmark and while in local production mode, we send via the django email client?

Reply:
David set up 3 postmark servers, for production, development and testing.  I think this will work well, so we can see in any case what message went out.